src/main/webapp/app/shared/shared.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { CheckboxComponent } from './components/checkbox/checkbox.component';
+import { MatCheckboxModule } from '@angular/material/checkbox';
+
+
+
+@NgModule({
+  declarations: [CheckboxComponent],
+  imports: [
+    CommonModule,
+    MatCheckboxModule
+  ],
+  exports: [CheckboxComponent]
+})
+export class SharedModule { }
```

Comment:
We do it the same here at coyo but of course with more than one shared module :)

Reply:
Thanks 👍